sre-recipes/recipes/configs_based/README.md

```diff
@@ -0,0 +1,128 @@
+# Configs-Based SRE Recipe
+
+This directory contains configs for supported config-based SRE Recipes.
+
+## Writing the Config
+
+Each SRE Recipe config is a YAML file with the following overall structure.
+
+```
+name: <Put the name of your recipe here. This is not user visible.>
+description: <Put the description of your recipe here. This is not user visible.>
+config:
+  # This section lists the configs for all the actions to perform for this SRE
+  # Recipe. In practice, this is where you try to break sandbox services.
+  #
+  # If your SRE Recipe has complex logic, prefer running a shell command to
+  # run a shell script instead.
+  #
+  # You can use, or contribute, more action templates by referring to the
+  # "Supported SRE Recipe Actions" below.
+  break:
+    # The `run` action template simply runs the given shell commands as it is.
+    - run: <shell command>
+
+    # The `loadgen` action template with `spawn` will spawn the given load shape
+    # producible by `user_type` (see `sre/loadgenerator/locust_tasks`) at a
+    # `spawn_rate` of users per second until it reaches a total of `user_count`
+    # users. The users will keep generating load until at least `stop_after`
+    # seconds have passed.
+    - loadgen: spawn
+      user_type: BasicHomePageViewingUser
+      user_count: 20
+      spawn_rate: 5
+      stop_after: 600
+
+    # The `loadgen` action template with `stop` value will stop any active
+    # load generation produced by SRE Recipes. It is ok to call this even if
+    # there is no load generation ongoing.
+    - loadgen: stop
+  # This section lists the configs for all the actions to perform for this SRE
+  # Recipe. In practice, this is where you try to restore sandbox services
+  # back to its original working condition.
+  #
+  # If your SRE Recipe has complex logic, prefer running a shell command to
+  # run a shell script instead.
+  #
+  # You can use, or contribute, more action templates by referring to the
+  # "Supported SRE Recipe Actions" below.
+  restore:
+    # The same set of action templates as `break` are supported in `restore`
+    - run: <shell command>
+    - loadgen: stop
+  hint: <Put the hint for your recipe here.>
```

Comment:
I think "Put the hint **string** for your recipe here" might be more clear, so it's explicit what the hint should look like

Reply:
Done.